File: src/main/scala/analysis/product/bean/AreaTop3Product.scala
package analysis.product.bean

/**
 *
 * @param taskid
 * @param area
 * @param areaLevel
 * @param productid
 * @param cityInfos
 * @param clickCount
 * @param productName
 * @param productStatus
 */
case class AreaTop3Product(taskid:String,
                           area:String,
                           areaLevel:String,
                           productid:Long,
                           cityInfos:String,
                           clickCount:Long,
                           productName:String,
                           productStatus:String)

File: src/main/java/com/jxust/entity/AreaTop3Product.java
package com.jxust.entity;

import java.io.Serializable;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


/**
* Create by Jayden in 2020/3/7
*/

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AreaTop3Product implements Serializable {
    private String taskid;

    private String area;

    private String arealevel;

    private Long productid;

    private String cityinfos;

    private Long clickcount;

    private String productname;

    private String productstatus;

    private static final long serialVersionUID = 1L;
}